vi/news/_posts/2015-04-13-ruby-2-0-0-p645-released.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: news_post
+title: "Phát hành Ruby 2.0.0-p645"
+author: "usa"
+translator: "My Mai"
+date: 2015-04-13 12:00:00 +0000
+lang: vi
+---
+
+Chúng tôi rất vui mừng thông báo việc phát hành phiên bản Ruby 2.0.0-p645.
+
+Lần phát hành này bao gồm việc sửa lỗi bảo mật cho tiện ích OpenSSL. Để biết thêm chi tiết xin vui lòng xem chủ đề dưới đây.
+
+* [CVE-2015-1855: Ruby OpenSSL Hostname Verification](https://www.ruby-lang.org/en/news/2015/04/13/ruby-openssl-hostname-matching-vulnerability/)
```

Comment:
The title should use Vietnamese: `Xác nhận Hostname Ruby OpenSSL` and should link to trasnlation: `https://www.ruby-lang.org/vi/news/2015/04/13/ruby-openssl-hostname-matching-vulnerability/`:

```
* [CVE-2015-1855: Xác nhận Hostname Ruby OpenSSL](https://www.ruby-lang.org/vi/news/2015/04/13/ruby-openssl-hostname-matching-vulnerability/)
```

:smile: